CHANGELOG.md

```diff
@@ -189,7 +192,9 @@ Initial release
 
 
 [Unreleased]:
-https://github.com/nasa/cumulus-cumulus-message-adapter-js/compare/v1.3.0...HEAD
+https://github.com/nasa/cumulus-cumulus-message-adapter-js/compare/v2.3.0...HEAD
```

Comment:
Looks like this hasn't been updated in a while. I can add all of the intermediate releases if we think it's necessary. I'm being lazy here because it doesn't seem super critical or confusing.